### DISCUSSION TITLE
Database query use parameters

### DISCUSSION THREAD

**Discussion by MrTianFei** [nextgenhealthcare/connect#4781] [Q&A] [answered]
I am a mirth novice and I tried to use SQL conditional queries but seemed to fail 
The problem has been solved, I made a mistake, the following code is correct  
![image](https://user-images.githubusercontent.com/86398277/137059959-623c16d4-b060-49cf-86ba-a48a9e85ba70.png)

**Comment by pacmano1** [comment, marked as answer]
Poster answered their own question however I think the problem was the syntax of a  "paramaterized" query that happens to be conditional.
